app/apps.json

```diff
@@ -205,5 +205,32 @@
       "type": "addon",
       "url": "https://github.com/xeonchen/fxos-line-fixer",
       "revision": "4c93314c2fa985aa41316f9423f06090dec059d4"
+    },
+    "https://lmorchard.github.io/fxos-addon-messages-bigger-send-button/manifest.webapp": {
```

Comment:
Sure, I can take a look at submitting a PR. Main reason I could see for not doing it is that I just kind of picked a size at random for the buttons, and the layout is a little cruddy & misaligned after. Could probably use some more expert UX tweaking